src/zap/zapcode.cpp

```diff
@@ -1202,6 +1202,49 @@ void ZapUnwindData::Save(ZapWriter * pZapWriter)
 #endif //REDHAWK
 }
 
+#elif defined(_TARGET_X86_)
+
+UINT ZapUnwindData::GetAlignment()
+{
+    return sizeof(ULONG);
+}
+
+DWORD ZapUnwindData::GetSize()
+{
+    DWORD dwSize = ZapBlob::GetSize();
+
+#ifndef REDHAWK
+    // Add space for personality routine, it must be 4-byte aligned.
+    // Everything in the UNWIND_INFO has already had its size included in size
+    dwSize = AlignUp(dwSize, sizeof(ULONG));
+
+    dwSize += sizeof(ULONG);
+#endif //REDHAWK
```

Comment:
While working on #8889, I overhear that x86 has no personality routine space (please refer to ``reservePersonalityRoutineSpace``), but I'm not sure whether the line should be aligned with that, or not.

@jkotas @janvorli Could you correct me if there is something wrong?